src/datepicker/__tests__/datepicker-range-highlight.scenario.js

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+// eslint-disable-next-line import/extensions
+import Screener, {Steps} from 'screener-storybook/src/screener';
+import {StatefulDatepicker} from '../index.js';
+
+export const name = 'datepicker-range-highlight';
+
+export const component = () => {
+  const rightArrow = '[aria-label="Next month"]';
+  return (
+    <Screener
+      steps={new Steps()
+        .wait(rightArrow)
```

Comment:
@jh3y this line times out in screener: 
`[Error] waitFor('[aria-label="Next month"]') timed-out` - probably the selector is off?